torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -258,6 +258,13 @@ void initNvFuserPythonBindings(PyObject* module) {
             return IrBuilder::create<Double>(val);
           },
           py::return_value_policy::reference)
+      .def(
```

Comment:
These changes will be gone once https://github.com/pytorch/pytorch/pull/80522 is merged.